src/Reflection/RepositoryCountByMethodReflection.php

```diff
@@ -0,0 +1,128 @@
+<?php declare(strict_types = 1);
+
+namespace SaschaEgerer\PhpstanTypo3\Reflection;
+
+use PHPStan\Reflection\ClassMemberReflection;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\FunctionVariant;
+use PHPStan\Reflection\MethodReflection;
+use PHPStan\TrinaryLogic;
+use PHPStan\Type\Generic\TemplateTypeMap;
+use PHPStan\Type\IntegerType;
+use PHPStan\Type\Type;
+
+class RepositoryCountByMethodReflection implements MethodReflection
+{
+
+	/** @var \PHPStan\Reflection\ClassReflection */
+	private $classReflection;
+
+	/** @var string */
+	private $name;
+
+	public function __construct(ClassReflection $classReflection, string $name)
+	{
+		$this->classReflection = $classReflection;
+		$this->name = $name;
+	}
+
+	public function getDeclaringClass(): ClassReflection
+	{
+		return $this->classReflection;
+	}
+
+	public function isStatic(): bool
+	{
+		return false;
+	}
+
+	public function isPrivate(): bool
+	{
+		return false;
+	}
+
+	public function isPublic(): bool
+	{
+		return true;
+	}
+
+	public function getPrototype(): ClassMemberReflection
+	{
+		return $this;
+	}
+
+	public function getName(): string
+	{
+		return $this->name;
+	}
+
+	/**
+	 * @return RepositoryFindByParameterReflection[]
+	 */
+	public function getParameters(): array
+	{
+		return [];
```

Comment:
Should return something like return 
`
[
     new  RepositoryFindByParameterReflection('argument', new MixedType())
];
`

otherwise the following error occurs:
'countByFoo() invoked with 1 parameters, 0 required'

Reply:
Thats true... I haven't used this in a long time and thought there was no argument for countBy but makes sense for sure ;)

Reply:
@sascha-egerer Works great. Ready to merge